apps/nextjs-website/react-components/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Container, Grid, Stack } from '@mui/material';
+import { AccordionItem } from './AccordionItem';
+import { Title, Subtitle, Body } from '../common/Common';
+import { AccordionProps } from '../../types/Accordion/Accordion.types';
+import {
+  BackgroundColorAlternative,
+  TextColor,
+} from '../common/Common.helpers';
+
+const Accordion = (props: AccordionProps) => {
+  const {
+    title,
+    subtitle,
+    description,
+    accordionItems,
+    theme,
+    layout = 'left',
+  } = props;
+
+  const textColor = TextColor(theme);
+  const backgroundColor = BackgroundColorAlternative(theme);
+  const isCenterLayout = layout === 'center';
+  const textAlignment = isCenterLayout ? 'center' : 'left';
+
+  return (
+    <Box
+      sx={{ py: { xs: 4, md: 10 } }}
+      bgcolor={backgroundColor}
+      component='section'
+    >
+      <Container>
+        <Grid spacing={{ xs: 5, md: isCenterLayout ? 7 : 16 }} container>
+          <Grid item xs={12} md={isCenterLayout ? 12 : 4}>
+            <Stack px={{ xs: 1, md: 0 }} spacing={{ xs: 2, md: 5 }}>
+              {/** Title */}
+
+              <Title
+                variant='h4'
+                textColor={textColor}
+                title={title}
+                textAlign={textAlignment}
+              />
+
+              {/** Subtitle */}
+
+              {subtitle && (
+                <>
+                  <Subtitle
+                    variant='h6'
+                    textColor={textColor}
+                    subtitle={subtitle}
+                    textAlign={textAlignment}
+                  />
+                </>
```

Comment:
```suggestion
                <Subtitle
                  variant='h6'
                  textColor={textColor}
                  subtitle={subtitle}
                  textAlign={textAlignment}
                />
```
No need for the fragment.